### PR TITLE
Add Scalingo PaaS

### PR DESCRIPTION
**Pull request by Ungredda**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ I'm a big fan of [_Awesome_ selfhosted](https://github.com/awesome-selfhosted/aw
 ### PaaS
 
 - [Clever cloud](https://www.clever-cloud.com/) - Clever Cloud is an IT Automation platform. We manage all the ops work while you focus on your business value :fr:.
+- [Scalingo](https://www.scalingo.com/) - Scalingo is a high-availability platform as a service (PaaS) with an expert support. We remove the pain of handling servers, application stacks, software dependencies and databases. We host more than 50 different technologies and support 6 different databases.
```
Comment (kakawait) [inline readme.md:65]:
```suggestion
- [Scalingo](https://www.scalingo.com/) - Scalingo is a high-availability platform as a service (PaaS) with an expert support. We remove the pain of handling servers, application stacks, software dependencies and databases. We host more than 50 different technologies and support 6 different databases :fr:.
```

**Review by kakawait — CHANGES_REQUESTED** [review]
Just add small flag for France 👍

File: readme.md (hunk 1 above)
Comment (Ungredda) [reply readme.md]: Indeed! Sorry done :)

Comment (kakawait) [reply readme.md]: @Ungredda sorry to be _psychorigid_ haha but can you put flag before the `.` to keep the same as other line :)

Comment (Ungredda) [reply readme.md]: Of course :) I think it should be done !

**Review by kakawait — APPROVED** [review]

**Pull request merged**